engine/src/main/java/org/terasology/world/chunks/blockdata/TeraDenseArray16Bit.java

```diff
@@ -68,6 +68,7 @@ public TeraArray deflate(TeraVisitingDeflator deflator) {
 
     @Override
     public int getEstimatedMemoryConsumptionInBytes() {
+        short[] data = this.data;
```

Comment:
I assume that you did this in the assumption that local variable access is faster than field instance access. As [indicated here on SO](http://stackoverflow.com/a/21614413/198397) there is no significant difference, so I'd assume that this is pointless.